src/wdio/services/ExpressDevServerService.js

```diff
@@ -0,0 +1,106 @@
+import express from 'express';
+import webpack from 'webpack';
+import MemoryFS from 'memory-fs';
+import mime from 'mime-types';
+import path from 'path';
+
+export default class ExpressDevServerService {
+  async onPrepare(config) {
+    if (!config.webpackConfig) {
+      // eslint-disable-next-line no-console
+      console.log('[ExpresDevService] No webpack configuration provided');
+      return;
+    }
+
+    const webpackConfig = config.webpackConfig;
+    const port = config.expressDevServer.port || 8080;
```

Comment:
It looks like the terra-core build failed due to not specifying the expressDevServer key within the wdio config.
We should update to be something like
```
const expressConfig = config.expressDevServer.port || 8080;
const port = expressConfig.port || 8080;
```

Reply:
Also, with the nightwatch tests passing and the wdio test passing it made realized we also need to update our nightwatch configuration. It uses the [WebpackDevServer service](https://github.com/cerner/terra-toolkit/blob/master/src/nightwatch/nightwatch.config.js#L24).

Reply:
The error is corrected, however I do not want to update the nightwatch test runner. We're deprecating nightwatch and would rather see tests converted to wdio, than switch nightwatch to the new test server.

Reply:
True. The more warning with the test setup will motive people to switch testing strategies.